service/src/main/java/uk/gov/hmcts/reform/fpl/model/ChildParty.java

```diff
@@ -47,6 +47,7 @@
     private final String litigationIssues;
     private final String litigationIssuesDetails;
     private final String showAddressInConfidentialTab;
+    private final String finalOrderIssued;
```

Comment:
any reason to set this in party rather than on the child object?

Reply:
All the properties which are displayed in the people on the case tab are stored in ChildParty. Is this not correct?

Reply:
As this flag pertains only to a child. I think it makes sense to have it here. Instead of Party which has subclasses - ApplicantParty, RespondentParty etc.

Reply:
Moved to Child object.